docs/INSTALLATION.md

```diff
@@ -1,65 +1,90 @@
 # Installation
 
-## Installing Stack
 
-If you don't have a recent Stack version installed in your system, follow the
-[installation instructions](http://docs.haskellstack.org/en/stable/install_and_upgrade/)
-for your operating system. After completing the installation, test that you
-have installed it correctly by running this command in a terminal:
+## Get GHCup
 
-```bash
-stack --version
-```
+The recommended way of installing Haskell tools is through [GHCup][ghcup].
 
-If it outputs a Stack version equal to or greater than 1.1.2, then you have it
-installed correctly.
+With GHCup you can manage your installed compilers, project/package managers, and language server.
 
-## Installing GHC
 
-Stack will automatically download and install GHC (the Glasgow Haskell
-compiler) when it is required. You can start the installation and test
-by running the following command in a terminal:
+## Install Stack
 
-```bash
-stack ghc
-```
+For doing Exercism exercises you only need Stack.
+
+To install Stack, simply run the GHCup installation command or script and follow the defaults but specify that you would like to have Stack installed as well.
+
+Did you miss the opportunity?
+No worries: just run `ghcup install stack` afterwards.
+
+GHCup will install
+
+- **A compiler** (GHC: the Glasgow Haskell Compiler)
+
+  Strictly speaking you do not need this for Exercism, but it comes in handy when you want to play with GHCi outside of exercise directories.
+
+- **Cabal**
+
+  A project/package manager.
+  You do not need this for Exercism.
+
+- **Stack**
 
-If this outputs something like `ghc: no input files`, GHC is installed
-correctly.
+  A project/package manager that you do need for Exercism.
```

Comment:
By being here, implies that we do need it, I think, if we do not mention things that we do not need, in this document.  I am not sure that this instills confidence in other places if we mention it is needed, and do not mention explicitly the need in other sections.

It is easier to state what is needed, and to state and then declare what is not needed.

Or perhaps put optional information in a complete section for those things that are nice but not necessary?